packages/virtualbox-deb/virtualbox-deb.pacscript

```diff
@@ -1,13 +1,36 @@
 name="virtualbox-deb"
-gives="virtualbox"
+gives="virtualbox-7.0"
 repology=("project: ${gives}")
-pkgver="7.0.14"
-url="https://download.virtualbox.org/virtualbox/${pkgver}/virtualbox-7.0_${pkgver}-161095~Ubuntu~jammy_amd64.deb"
+pkgver="7.0.16"
+arch=('amd64')
 breaks=("${gives}-bin")
 pkgdesc="VirtualBox is a general-purpose full virtualizer for x86 hardware, targeted at server, desktop and embedded use"
-hash="43aa0932f73433e7a6b450a82708d275ee5f2c98375cfa708a46ecf75b3d4edc"
-arch=('amd64')
+distro_version= "$(lsb_release -is)~$(lsb_release -cs)"
```

Comment:
Firstly, there is a space there which shouldn't on line 8. Next, using `$DISTRO` which is already provided instead of doing, this.

Reply:
I used `"$(lsb_release -is)~$(lsb_release -cs)"` instead of `$DISTRO` the download links' URLs follow this pattern.
I should change it anyway?

Reply:
Yes.

Reply:
> I used `"$(lsb_release -is)~$(lsb_release -cs)"` instead of `$DISTRO` the download links' URLs follow this pattern.
> I should change it anyway?

Yes. It's a little painful for the syntax and built-in commands etc.

But we need to respect the maximum possible to others pacscripts. 

(And after, it's a pleasure to work ^^)

Reply:
Oks, done.